src/Essentials/src/Permissions/Permissions.windows.cs

```diff
@@ -173,6 +174,56 @@ public partial class Media : BasePlatformPermission
 
 		public partial class Microphone : BasePlatformPermission
 		{
+			/// <inheritdoc/>
+			protected override Func<IEnumerable<string>> RequiredDeclarations => () => ["microphone"];
+
+			/// <inheritdoc/>
+			public override Task<PermissionStatus> CheckStatusAsync()
+			{
+				EnsureDeclared();
+				return Task.FromResult(CheckStatus() switch
+				{
+					DeviceAccessStatus.Allowed => PermissionStatus.Granted,
+					DeviceAccessStatus.DeniedBySystem => PermissionStatus.Denied,
+					DeviceAccessStatus.DeniedByUser => PermissionStatus.Denied,
+					_ => PermissionStatus.Unknown,
+				});
+			}
+
+			/// <inheritdoc/>
+			public override async Task<PermissionStatus> RequestAsync()
+			{
+				EnsureDeclared();
+
+				// If already explicitly allowed, return that
+				var status = CheckStatus();
+				if (status == DeviceAccessStatus.Allowed)
+					return PermissionStatus.Granted;
+
+				try
+				{
+					var settings = new MediaCaptureInitializationSettings
+					{
+						StreamingCaptureMode = StreamingCaptureMode.Audio
+					};
+
+					var mediaCapture = new MediaCapture();
+					await mediaCapture.InitializeAsync(settings);
+					mediaCapture.Dispose();
```

Comment:
The `MediaCapture` object isn't properly disposed if `InitializeAsync` throws an exception. Should not happen, but we could cover that case too.

Reply:
Could you use a using block?